src/elm/Transaction/Summary.elm

```diff
@@ -177,7 +183,30 @@ type RejectReason
     | NotAllowedToReceiveEncrypted
       -- |The account is not allowed to send encrypted transfers (or transfer from/to public to/from encrypted)
     | NotAllowedToHandleEncrypted
-
+    -- |A configure baker transaction is missing one or more arguments in order to add a baker.
+    | MissingBakerAddParameters
+    -- |A configure baker transaction to remove baker is passed unexpected arguments.
+    | UnexpectedBakerRemoveParameters
+    -- |Not all baker commissions are within allowed ranges
+    | CommissionsNotInRangeForBaking
+    -- |Tried to add baker for an account that already has a delegator
+    | AlreadyADelegator
+    -- |The amount on the account was insufficient to cover the proposed stake
+    | InsufficientBalanceForDelegationStake
+    -- |A configure delegation transaction is missing one or more arguments in order to add a delegator.
+    | MissingDelegationAddParameters
+    -- |A configure delegation transaction to remove delegation is passed unexpected arguments.
+    | UnexpectedDelegationRemoveParameters
+    -- |The change could not be made because the delegator is in cooldown
+    | DelegatorInCooldown
+    -- |Account is not a delegation account
+    | NotADelegator T.AccountAddress
+    -- |Delegation target is not a baker
+    | DelegationTargetNotABaker T.BakerId
+    -- |The amount would result in pool capital higher than the maximum threshold
+    | StakeOverMaximumThresholdForPool
+    -- |The amount would result in pool with a too high fraction of delegated capital.
+    | PoolWouldBecomeOverDelegated    
```

Comment:
I see that `PoolClosed` is not included here. Is that on purpose?

Reply:
It's not on purpose. `PoolClosed` was added consensus-side a week after the initial support for the P4 reject reasons was added in the dashboard. The same holds for `InsufficientDelegationStake` and perhaps others too. I'll double-check.